dataland-frontend/tests/e2e/fixtures/frameworks/sme/SmePreparedFixtures.ts

```diff
@@ -10,67 +14,44 @@ import { generateNaceCodes } from "@e2e/fixtures/common/NaceCodeFixtures";
  */
-function manipulateFixtureForYearWithMultipleSectors(input: FixtureData<SmeData>, year: string): FixtureData<SmeData> {
-  input.companyInformation.companyName = "SME-year-" + year;
-  input.reportingPeriod = year;
-  input.t.general.basicInformation.sector = generateNaceCodes(2);
+function manipulateFixtureForSpecificTest(input: FixtureData<SmeData>): FixtureData<SmeData> {
+  const reportingPeriod = "2023";
+  input.companyInformation.companyName = "SME-year-" + reportingPeriod;
+  input.reportingPeriod = reportingPeriod;
+  input.t.general.basicInformation.sectors = generateNaceCodes(2);
   input.t.power ??= {};
   input.t.power.investments ??= {};
-  input.t.power.investments.percentageRangeForInvestmentsInEnhancingEnergyEfficiency = "LessThan1";
+  input.t.power.investments.investmentsInEnhancingEnergyEfficiency =
+    SmePowerInvestmentsInvestmentsInEnhancingEnergyEfficiencyOptions.LessThan1Percent;
   input.t.power.consumption ??= {};
-  input.t.power.consumption.percentageRangeForEnergyConsumptionCoveredByOwnRenewablePowerGeneration = "LessThan25";
-  input.t.general.companyFinancials = {
-    revenueInEUR: 0,
-    operatingCostInEUR: 1000000,
-    capitalAssetsInEUR: 2000000,
+  input.t.power.consumption.energyConsumptionCoveredByOwnRenewablePowerGeneration =
+    SmePowerConsumptionEnergyConsumptionCoveredByOwnRenewablePowerGenerationOptions.LessThan25Percent;
+  input.t.general.financialInformation = {
+    revenueInEur: 2500000,
+    operatingCostInEur: 1000000,
+    capitalAssetsInEur: 10000000,
   };
   return input;
 }
 
 /**
- * Sets the company name and headquarters address with maximum number of address fields
+ * Sets the company name to a specific value to be able to pick this dataset from the prepared fixtures.
  * @param input Fixture data to be manipulated
  * @returns the manipulated fixture data
  */
-function manipulateFixtureForMaximumAddress(input: FixtureData<SmeData>): FixtureData<SmeData> {
-  input.companyInformation.companyName = "SME-maximum-address";
-  input.reportingPeriod = "2021";
-  input.t.general.basicInformation.addressOfHeadquarters = {
-    streetAndHouseNumber: "Main Street 12",
-    postalCode: "12345",
-    city: "Nonexistingen",
-    state: "Fiction",
-    country: "Imagination",
-  };
-  return input;
-}
-
-/**
- * Sets the company name and headquarters address with minimum number of address fields
- * @param input Fixture data to be manipulated
- * @returns the manipulated fixture data
- */
-function manipulateFixtureForMinimumAddress(input: FixtureData<SmeData>): FixtureData<SmeData> {
-  input.companyInformation.companyName = "SME-minimum-address";
-  input.reportingPeriod = "2022";
-  input.t.general.basicInformation.addressOfHeadquarters = {
-    streetAndHouseNumber: null,
-    postalCode: null,
-    city: "City 17",
-    state: null,
-    country: "Uninspired",
-  };
+function manipulateFixtureForNoNullFields(input: FixtureData<SmeData>): FixtureData<SmeData> {
```

Comment:
Is the function name truthful?

Reply:
Nicht so ganz präzise.
Habe es präziser benamt und die doc auch etwas erweitert.